source/Jobbr.Server.ForkedExecution.Tests/Jobbr.Server.ForkedExecution.Tests.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Do you know the difference between Sdk and Sdk.Web?

Reply:
I think this is a genuine mistake. I first thought that you need to do this to get the web application hosting stuff to work in a class library, but in the end you can just include a "Microsoft.AspNetCore.Mvc.Core" NuGet if I recall correctly. 

Probably not even necessary in the test project anyway.